code/game/gamemodes/colonialmarines/bioscan.dm

```diff
@@ -0,0 +1,30 @@
+/proc/xeno_scan()
+	var/pInStation = 0
+	var/pInPlanet = 0
+	for(var/mob/G in player_list)
+		var/turf/playerturf = get_turf(G)
+		if(playerturf.z == ZLEVEL_STATION && ishuman(G))
+			pInStation ++
+		else if(ishuman(G))

```

Comment:
what if they're on centcom or some other zlevel that might be made in future?strict-check this with playerturf.z == ZLEVEL_PLANET please
also i think i made a global list that has only humans, perhaps use that to avoid checking ishuman so many times